models/team-member.ini

```diff
@@ -38,3 +38,10 @@ width = 1/4
 [fields.description]
 label = Description
 type = rst
+
+[fields.superpower]
+label = Superpowers
+width = 1/2
+type = checkboxes
+choices = bdfn, batavia, voc, lektor, toga, android
+choice_labels = Beelevolent Dictator (For Now), Batavia, VOC, Lektor, Toga, Android
```

Comment:
Might want to add iOS to this list (although he's not on the list, David deals with Rubicon)

Not sure if Lektor belongs - I take it you were hinting at "can maintain the website"?